boards/arm/nrf54h20pdk_nrf54h20/nrf54h20pdk_nrf54h20_cpuapp_defconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF54HX=y
+CONFIG_SOC_NRF54H20=y
+CONFIG_SOC_NRF54H20_ENGA_CPUAPP=y
+CONFIG_BOARD_NRF54H20PDK_NRF54H20_CPUAPP=y
+
+CONFIG_USE_DT_CODE_PARTITION=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable cache
```

Comment:
```suggestion
# Enable instruction cache
```

Reply:
Same in Radio core.

Reply:
I moved disabling of DCACHE to `Kconfig.defconfig`, so that it can be overridden if needed. Please, take a look.